src/core/style/expressions/viewportAggregation.js

```diff
@@ -39,3 +39,48 @@ function generateViewportExpression(metadataPropertyName) {
         }
     };
 }
+
+
+export class ViewportPercentile extends Expression {
```

Comment:
should this be moved to its own file?

Reply:
I don't think so, it is just a specialization of the other viewport aggregation functions.

Reply:
🤔 what about creating a viewportAggregation folder with files in it?

Reply:
Too much noise for a 86 lines long file. And this pattern of having multiple classes of the same type like binary operations are in other places too, with a similar, if not less, line count.